src/transformState.js

```diff
@@ -5,7 +5,33 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key in actions) {
```

Comment:
[Checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 1st item

Reply:
> [Checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 1st item

But if I have an array of objects, why I have to iterate through for...of instead of for...in?